install.rb

```diff
@@ -194,6 +205,9 @@ def prepare_installation
     opts.on('--bindir[=OPTIONAL]', 'Installation directory for binaries', 'overrides RbConfig::CONFIG["bindir"]') do |bindir|
       InstallOptions.bindir = bindir
     end
+    opts.on('--localedir[=OPTIONAL]', 'Installation directory for locale information', 'Default /opt/puppetlabs/puppet/share/puppet/locale') do |localedir|
```

Comment:
Comparing to `localedir` on line 321, is there an extra 'puppet' in the `Default` path here?